src/main/scala/temple/builder/OrchestrationBuilder.scala

```diff
@@ -0,0 +1,41 @@
+package temple.builder
+
+import temple.ast.Metadata.Database
+import temple.ast.{Metadata, ServiceBlock}
+import temple.builder.project.ProjectConfig
+import temple.generate.kube.ast.OrchestrationType.{OrchestrationRoot, Service}
+import temple.generate.kube.ast.gen.LifecycleCommand
+
+object OrchestrationBuilder {
+
+  def createServiceOrchestrationRoot(
+    projectName: String,
+    services: Seq[(String, ServiceBlock, Int)],
+  ): OrchestrationRoot =
+    OrchestrationRoot(
+      services map {
+        case (name, service, port) =>
+          val dockerImage = s"temple-$projectName-$name"
```

Comment:
Not sure I like adding `temple` to the start of each Docker image - feels like it ties it too closely with generated code? @xsanda @smithwjv?

Reply:
Is there a reason we didn’t just go for `s"$projectName-$name"`? @Lewiky @jaylees14